vnext/DeforkingPatches/.clang-format

```diff
@@ -0,0 +1,91 @@
+---
+AccessModifierOffset: -1
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: false
+AlignConsecutiveDeclarations: false
+AlignEscapedNewlinesLeft: true
+AlignOperands:   false
+AlignTrailingComments: false
+AllowAllParametersOfDeclarationOnNextLine: false
+AllowShortBlocksOnASingleLine: false
+AllowShortCaseLabelsOnASingleLine: false
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortIfStatementsOnASingleLine: false
+AllowShortLoopsOnASingleLine: false
+AlwaysBreakAfterReturnType: None
+AlwaysBreakBeforeMultilineStrings: true
+AlwaysBreakTemplateDeclarations: true
+BinPackArguments: false
+BinPackParameters: false
+BraceWrapping:
+  AfterClass:      false
+  AfterControlStatement: false
+  AfterEnum:       false
+  AfterFunction:   false
+  AfterNamespace:  false
+  AfterObjCDeclaration: false
+  AfterStruct:     false
+  AfterUnion:      false
+  BeforeCatch:     false
+  BeforeElse:      false
+  IndentBraces:    false
+BreakBeforeBinaryOperators: None
+BreakBeforeBraces: Attach
+BreakBeforeTernaryOperators: true
+BreakConstructorInitializersBeforeComma: false
+BreakAfterJavaFieldAnnotations: false
+BreakStringLiterals: false
+ColumnLimit:       80
```

Comment:
Hmm... Actual Facebook React Native code doesn't' seem to follow this. I wonder what's going on here.

Reply:
Which bit do they not follow?
I opened #3883 as something we might want to think about.

Reply:
The stock facebook files seem to align line length to 120 characters. There were some other changes that got applied, like moving pointer asterisks to hug the variable name instead of type name.

Reply:
Actually, it looks like Facebook just didn't run clang-format on this file until last October. I can see there's a CircleCI requirement for JS formatting but not sure how well formatted their native codebase is. https://github.com/facebook/react-native/commit/ba18ee9b87a35fccec4385a67d393ab4a09f5b29